### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by BANCUNGUYE66**
I added a responsive desktop version for the portfolio. I Used media queries in Css file (min-width: 768px).

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -20,6 +20,17 @@
...
+                 <div>
+                  About
+                 </div>
+                 <div>
+                  Contact
+                 </div>
+             </div>
           </div>
       </nav>
    </header>
```
Comment (Shisui6) [inline index.html:36]: Kindly update the styling of your headline section to better match the figma design🙏.

Your design:
image

Figma design:
image

A few things to note:
- Please use the correct `font-style` `font-size` and `font-weight` for all the text elements. You can click on each element in the figma template and see the styles applied on the inspect panel on the right. I believe the correct font should be `poppins`
- Please put a line break `<br>` between `I'm Aima` and `Glad to see you!`

File: index.html [hunk 2]
```diff
@@ -48,134 +59,144 @@ <h1>I’m Aima Glad to see you! </h1>
...
+                     <div class="skills_button">
+                        <ul>
+                           <li><a href="#">html</a></li>
+                           <li><a href="#">css</a></li>
+                           <li><a href="#">javascript</a></li>
+                        </ul>
+                     </div>
+                     <div class="Project_button">
+                        <button type="button"> See Project </button>
+                     </div>
```
Comment (Shisui6) [inline index.html:88]: Kindly update the styling of each of your cards to better match the figma design🙏.

Your design:
image

Figma design:
image

A few things to note:
- Kindly increase the height of each card
- Please increase the left and right margin of each card. Currently your cards are overstretched
- Kindly reduce the left and right padding for each card. 
- Please use the correct `font-style` `font-size` and `font-weight` for each card's text elements. You can click on each element in the figma template and see the styles applied on the inspect panel on the right. I believe the correct font should be `poppins`
- Kindly add button interaction styling to all your cards when the buttons are hovered and pressed. You can do this by using the `:hover` and `:active` pseudo selectors. Please follow the button styling guide provided by figma
image

File: index.html [hunk 3]
```diff
@@ -48,134 +59,144 @@ <h1>I’m Aima Glad to see you! </h1>
...
-                  Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
-               </p>
+               <div class="about_p">
+                  <p>
+                     Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                  </p>
+               </div>
                <div class="connect">
                    <a href="#"> LET’S CONNECT</a>
                </div>
```
Comment (Shisui6) [inline index.html:202]: Kindly update the styling of your about section to better match the figma design🙏.

Your design:
image

Figma design:
image

A few things to note:
- Please put a line break `<br>` between `About` and `Myself`
- Please use the correct `font-style` `font-size` and `font-weight` for all the text elements. You can click on each element in the figma template and see the styles applied on the inspect panel on the right. I believe the correct font should be `poppins`
- Kindly increase the top and bottom padding for the technology cards(HTML, CSS, JS) and increase the spacing between the image and the text
- Kindly increase the top and bottom padding for the `frameworks` and `skills` dropdown sections
- Kindly add button interaction styling to the `get my resume` button. Please follow the button styling guide provided by figma
image

File: index.html [hunk 4]
```diff
@@ -209,22 +230,25 @@ <h2>About Myself </h2>
...
             </div>
          </section>
          <section class="contact_page">
             <div class="contact_page_container">
                <h2><span>Contact me</span></h2>
-               <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>   
+               <p><span>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it</span> </p>   
             </div>
             <form class="contact_form" action="https://formspree.io/f/xyyawbjz" method="post">
                 <ul class="li_st">
```
Comment (Shisui6) [inline index.html:254]: Kindly update the styling of your contact section to better match the figma design🙏.

Your design:
image

Figma design:
image

A few things to note:
- Please increase the width of the main content of the contact section
- Please use the correct `font-style` `font-size` and `font-weight` for all the text elements. You can click on each element in the figma template and see the styles applied on the inspect panel on the right. I believe the correct font should be `poppins`

**Review by Shisui6 — CHANGES_REQUESTED** [review]
Hi @BANCUNGUYE66, 

# Requires Changes 🍀
You have done a great job so far!
Your design is good 🖼️, and your code follows best practices. However, there are a few issues that you still need to work on🪲.

## Highlights
- Good design ✅
- Good use of media queries ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

You can also consider the following:

N/A

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Favourezeugwa — APPROVED** [review]
## STATUS: APPROVED 🥇 🏆 🍹 

Hi @BANCUNGUYE66 ,

I am Favour 👩🏽 , your reviewer for this PR.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Yayyy! 💃🏽, Congratulations! 🎉 🎉🎉

happy

## Highlights 💯 
- No Linter errors ✔️
- Good commit messages ✔️
- Professional Readme file ✔️
- Descriptive PR ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**